README.md

```diff
@@ -244,14 +244,11 @@ Following inputs can be used as `step.with` keys
 
 Following outputs are available
 
-| Name          | Type    | Description                           |
-|---------------|---------|---------------------------------------|
-| `name`        | String  | Builder name |
-| `driver`      | String  | Builder driver |
-| `endpoint`    | String  | Builder node endpoint |
-| `status`      | String  | Builder node status |
-| `flags`       | String  | Builder node flags (if applicable) |
-| `platforms`   | String  | Builder node platforms available (comma separated) |
+| Name        | Type   | Description            |
+|-------------|--------|------------------------|
+| `name`      | String | Builder name           |
+| `driver`    | String | Builder driver         |
+| `nodes`     | JSON   | Builder nodes metadata |
```

Comment:
We should describe the JSON schema of the `nodes` metadata, maybe in a table below?